Tasks/AzureVmssDeployment/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "AzureVmssDeployment",
```

Comment:
I have not added shrinkwrap. I started feeling that it was causing more confusion rather than good. it overrides the modules' version specified in package.json. Plus it is a maintenance issue as well to continuously update it. Very few tasks are using shrinkwrap.